src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
+
+public class Square extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line before the class declaration.